src/XenGnttab.cpp

```diff
@@ -228,15 +241,24 @@ XenGnttabDmaBufferImporter::~XenGnttabDmaBufferImporter()
 
 void XenGnttabDmaBufferImporter::init(domid_t domId, int fd, GrantRefs &refs)
 {
+#ifdef GNTTAB_HAS_DMABUF_OFFSET
+	uint32_t offset;
```

Comment:
size_t would be better.

Reply:
```
	ret = xengnttab_dmabuf_imp_to_refs_v2(mHandle, domId, fd,
										  refs.size(), refs.data(),
										  &offset);

```
accepts uint32_t as offset, so I think I should keep it as is

Reply:
```
libxenbe/src/XenGnttab.cpp:259:20: error: cannot convert 'size_t* {aka long unsigned int*}' to 'uint32_t* {aka unsigned int*}' for argument '6' to 'int xengnttab_dmabuf_imp_to_refs_v2(xengnttab_handle*, uint32_t, uint32_t, uint32_t, uint32_t*, uint32_t*)'
             &offset);
                    ^

```

Reply:
ok